elytron/src/test/resources/org/wildfly/extension/elytron/tls-test.xml

```diff
@@ -14,8 +14,10 @@
         </filesystem-realm>
     </security-realms>
     <credential-stores>
-        <credential-store name="test" case-sensitive="false">
-            <uri>cr-store://test/target/tlstest.keystore</uri>
+        <credential-store name="test" case-sensitive="false" location="target/tlstest.keystore">
```

Comment:
Just to be sure the parsing and marshalling works properly, I'd like to see the `create` and `modifiable` attributes used in the test xml as well.